origin.spec

```diff
@@ -103,6 +414,7 @@ Obsoletes:      openshift-clients-redistributable < %{package_refector_version}
 
 %description clients-redistributable
 %{summary}
+%endif
```

Comment:
Hmm, ok. I had hoped that at some point the console would serve up the clients but that hasn't caught on yet. If it does happen we'll probably want to reconsider this as it would be the content downloaded via the console.

Right now the immediate term intention is just to provide oc access via a terminal in the web console. Origin users can of course download the binaries from github as well.